app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -204,6 +204,11 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
                 super.dispatchKeyEvent(event)
     }
 
+    private fun onNoActiveSession() {
```

Comment:
nit: you can define this in the session observer (for clearer encapsulation of who wants to call this)

Reply:
I'll update.